posts/2022-01-14-construct-vendia-univseral-application-using-domain-driven-design.md

```diff
@@ -0,0 +1,177 @@
+# Construct Vendia Universal Application using Domain Driven Design
```

Comment:
Can you work with TZ and Caroll on the title?  Something like "Applying Domain Driven Design to Blockchains" or "4 DDD Patterns for your Next Distributed Ledger".

Reply:
I will resolve all the changes requested, take an export of blog readme and send it over to TZ and Carrol to suggest a title.